### DISCUSSION TITLE
Way to create a modal window (like the Link Map window) with frontend API?

### DISCUSSION THREAD

**Discussion by MorganeLeCaignec** [TriliumNext/Trilium#1596] [Q&A] [answered]
Is there a way to call the creation of a modal window appearing on top of the application (like the one appearing when clicking on the "Link Map" button in the right sidebar) through a code block ?

I'd be useful to display custom widget notes in this way, such as relation maps or (in my case) a timeline display.

**Comment by zadam** [comment, marked as answer]
Hi, there's no specific Trilium API for that but you can use what Trilium uses - [Bootstrap modals](https://getbootstrap.com/docs/4.5/components/modal/)
